config.go

```diff
@@ -1090,13 +1083,6 @@ func parseRPCParams(cConfig *chainConfig, nodeConfig interface{}, net chainCode,
 		case litecoinChain:
 			daemonName = "litecoind"
 		}
-		// If only one or two of the parameters are set, we assume the
-		// user did that unintentionally.
-		if conf.RPCUser != "" || conf.RPCPass != "" || conf.ZMQPath != "" {
```

Comment:
We should instead have a check at the end of the method, ensuring the parameters are all properly set before returning (either from command line or extracted from the config).